### PR TITLE
Update variational inference notebook with sensible defaults to work of-the-box

### PR DESCRIPTION
**Pull request by rothn**
Currently, the training notebook does not run out-of-the-box after running the data ingestion notebook. A minimal set of modifications did not produce a good model, so I also performed some minor tuning to help ensure a good out-of-box experience for anyone who tries this.

Changelist:
* Changed input paths to match the "_demo" files that will automatically be created by the latest data_ingestion.ipynb
* Add xavier initialization (network was not initialized, and doing so is considered a best practice)
* Changed patience from 2000 to 30 to avoid overtraining
* Updated arguments to raster.generate_isoscapes_from_variational_model() to match latest code in ddf_common
* Re-introduced reference cell outputs

Reference Performance (splits from https://github.com/tnc-br/ddf_common/pull/46):
Val loss: 1.397790789604187
Train loss: 0.9393936395645142
Test loss: 0.9468271732330322
dO18 RMSE: 0.9135236042418461

### CODE REVIEW

**Comment by VimWizardVersace** [comment]
I noticed lower patience undertrained when doing distribution sampling. Looking at the loss graphs in this training run, I feel like it could keep improving.  

I agree 2000 is too much, but 30 is too little. Change it to 100?

**Comment by rothn** [comment]
> I noticed lower patience undertrained when doing distribution sampling. Looking at the loss graphs in this training run, I feel like it could keep improving.
> 
> I agree 2000 is too much, but 30 is too little. Change it to 100?

Sounds good. Agreed that it's kind of low, and 100 might be better especially based on the graph like you say

**Review by VimWizardVersace — DISMISSED** [review]

**Comment by VimWizardVersace** [comment]
Approved conditional on the patience level change

**Review by VimWizardVersace — APPROVED** [review]
